src/app/contribute/overview/guidelines/index.html

```diff
@@ -0,0 +1,102 @@
+<stache
+  pageTitle="Contribution guidelines"
+  navTitle="Guidelines"
+  showTableOfContents="true"
+  layout="sidebar"
+  navOrder="4">
+
+  <stache-page-summary>
+    <p>
+      We highly encourage contributions from all SKY UX users, and we ask that you follow these general guidelines so we can review and accept your contributions in a timely fashion.
+    </p>
+  </stache-page-summary>
+
+  <stache-page-anchor>
+    File an issue
+  </stache-page-anchor>
+  <p>
+    Before you start any work, <a stacheRouterLink="/contribute/overview/file-issue">file an issue</a>. The sooner we discuss potential code changes, the better.
+  </p>
+
+  <stache-page-anchor>
+    Accessibility tests
+  </stache-page-anchor>
+  <p>
+    All new components and changes to existing components require passing accessibility tests. This includes automated tests to run during visual regression tests as well as code review and manual keyboard tests. For more information about accessibility tests, see the <a stacheRouterLink="/learn/accessibility">accessibility section</a> and the <a stacheRouterLink="/learn/reference/tests" fragment="accessibility-tests">testing reference</a>.
+  </p>
+
+  <stache-page-anchor>
+    API changes
+  </stache-page-anchor>
+  <p>
+    For API changes that affect inputs and outputs for a component, we recommend that you provide your potential (not working) documentation and demo to the SKY UX team for review. This lets us head off API issues before implementation. Contributors should keep in mind that the API can change in the course of development.
```

Comment:
Can we switch out "head off" to something like "review" or "address"?

Reply:
Good call. I'll change it to "address"